t/22-dashboard.t

```diff
@@ -194,6 +194,7 @@ for my $url (@urls) {
 # parent group overview
 $t->get_ok('/parent_group_overview/' . $test_parent->id)->status_is(200);
 check_test_parent('expanded');
+$t->element_exists('#test_result_overview_link');
```

Comment:
Of course we should check whether the link is actually correct.

Reply:
Yes, this is under development yet